### ISSUE TITLE
Token settings not editable in gnome 40

### ISSUE THREAD

**Issue by dark-saga** [cinatic/gitlab-extension#9]
Hi. I'm on gnome 40 and I installed extension again after it was updated.

I revoked my old token and it seems that I'm not able to change (edit it) token to the new one in settings or to delete it. New checkbox "Only Owned Projects" works btw (i can check and uncheck it). I tried to remove extension and install it again, old token is still there.

Can you help me to change the token? :)

My setup. Arch Linux, gnome 40.

**Comment by cinatic** [comment]
Hi i was not able to reproduce it. Could you please dump the config via dconf and post the structure (replace all the sensitive fields before =))

`dconf read /org/gnome/shell/extensions/gitlab/gitlab-accounts | base64 -d`

also please tell me if `dconf read /org/gnome/shell/extensions/gitlab/gitlab-token` is empty

**Comment by dark-saga** [comment]
>Could you please dump the config via dconf and post the structure (replace all the sensitive fields before =))

>dconf read /org/gnome/shell/extensions/gitlab/gitlab-accounts | base64 -d
>base64: invalid input

Maybe I'm doing something wrong. Usually I don't have to mess with things in dconf

So, in /org/gnome/shell/extensions/gitlab 2 fields are present gitlab-accounts (which looks like hash of a token) and 
position-in-panel. Where is no gitlab-token string and value

So I tried (not good move) to put plain token in field gitlab-accounts. It didn't work, but old token disappeared from settings and I was able to see "+" button. Added new token and extension works now ) Thank you. Should I close the issue?
